test/futures-coinm/private.test.ts

```diff
@@ -0,0 +1,40 @@
+import { CoinMClient } from '../../src';
+import { CoinMSymbolOrderBookTicker } from '../../src/types/coin';
+
+describe('Private Futures USDM REST API Endpoints', () => {
+  const API_KEY = process.env.API_KEY_COM;
+  const API_SECRET = process.env.API_SECRET_COM;
+
+  const api = new CoinMClient({
+    disableTimeSync: true,
+    api_key: API_KEY,
+    api_secret: API_SECRET,
+  });
+  let book: CoinMSymbolOrderBookTicker[];
+
+  beforeEach(async () => {
+    if (!book) {
+      book = await api.getSymbolOrderBookTicker();
+    }
+  });
```

Comment:
Curiously, what was the plan for this? I only see it being used by one test and it's an extra API call per test in this file.

Reply:
ah I see, of course. There's also a beforeAll function you could use so you don't need to check for `book`

Reply:
Oh! great!! i'll use it.... TDD... getting to know it.

Reply:
changed in the new commit !